packages/dotnet-sdk/src/TeamsFx/TeamsUserCredential.cs

```diff
@@ -28,6 +28,8 @@ protected internal class AccessTokenJS
         /// Constructor of TeamsUserCredential.
         /// Developer need to call LoadConfigurationAsync(config) before using this class.
         /// </summary>
+        /// <exception cref="ExceptionCode.InvalidConfiguration">When client id, initiate login endpoint or simple auth endpoint is not found in config.</exception>
+        /// <exception cref="ExceptionCode.RuntimeNotSupported">When runtime is nodeJS.</exception>
```

Comment:
This could be changed to not in browser.

Reply:
got it